hopp/simulation/technologies/wind/pysam_wind.py

```diff
@@ -0,0 +1,65 @@
+from attrs import define, field
+
+import PySAM.Windpower as Windpower
+
+from hopp.simulation.base import BaseClass
+from hopp.simulation.technologies.sites import SiteInfo
+from hopp.type_dec import resource_file_converter
+from hopp.utilities.utilities import load_yaml
+
+
+@define
+class PySAMWind(BaseClass):
+    config_dict: dict = field(converter=dict)
+    site: SiteInfo = field()
+    timestep: tuple = field(default=(), converter=tuple)
+
+    system_model: Windpower = field(init=False)
+
+    def __attrs_post_init__(self):
+        input_file_path = resource_file_converter(self.config_dict["simulation_input_file"])
+        filetype = input_file_path.suffix.strip(".")
+        if filetype.lower() in ("yml", "yaml"):
+            input_dict = load_yaml(input_file_path)
+        else:
+            raise ValueError("Supported import filetype is YAML")
+
+        self.system_model = Windpower.new()
+        self.system_model.assign(input_dict)
+
+        self.wind_turbine_rotor_diameter = input_dict['Turbine']['wind_turbine_rotor_diameter']
+        self.wind_farm_xCoordinates = input_dict['Farm']['wind_farm_xCoordinates']
+        self.wind_farm_yCoordinates = input_dict['Farm']['wind_farm_yCoordinates']
+        self.nTurbs = len(self.wind_farm_xCoordinates)
```

Comment:
prefer `n_turbs` if possible

Reply:
Yes, barring any issues with internal names to PySAM.

Reply:
Look at extra attributes on pysam_wind and see if they are still needed after updating inheritance to powersource. If so, then figure out disconnect from system_model to _system_model.